test/jobs/benchmark_regression_job_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class BenchmarkRegressionJobTest < ActiveJob::TestCase
+  setup do
+    benchmark_result_type = create(:benchmark_result_type)
+    benchmark_type = create(:benchmark_type)
+    repo = create(:repo)
+    10.times do
+      commit = create(:commit, repo: repo)
+      @benchmark_run = create(:commit_benchmark_run,
+        benchmark_result_type: benchmark_result_type,
+        benchmark_type: benchmark_type,
+        initiator: commit
+      )
+    end
+  end
+
+  test "benchmark regression detected" do
+    @benchmark_run.update(result: {"sometime"=>"5.1"})
+    travel_to "2016-06-03 13:50:41 +0530" do
+      VCR.use_cassette('benchmark_regression') do
+        assert_equal ["sometime"], BenchmarkRegressionJob.new.perform(@benchmark_run.id)
+      end
```

Comment:
Actually just restore the previous unit tests. I realized the trouble of writing integration test isn't worth it.